### ISSUE TITLE
[TF2][Feature Request] Allow the Caber to taunt kill

### ISSUE THREAD

**Issue by Dee-Kay47** [ValveSoftware/Source-1-Games#4331]
It looks like it could taunt kill, I see no reason as to why it shouldn't. He literally swings it into the air.

**Comment by condor00fr** [comment]
Yeah, the animation clearly shows a tauntkill, they must have forgot to implement that.

**Comment by condor00fr** [comment]
I wonder just how insane would CaberKnights become if this ever gets added in.... Probably a lot, wich is a good thing, being an occasional CaberKnight myself I could really see this becoming viral.
